package-lock.json

```diff
@@ -2216,9 +2216,9 @@
       "integrity": "sha1-Ywn04OX6kT7BxpMHrjZLSzd8nns="
     },
     "picomatch": {
-      "version": "2.2.1",
-      "resolved": "http://verdaccio.internal.utterberry.com/picomatch/-/picomatch-2.2.1.tgz",
```

Comment:
verdaccio.internal.utterberry.com? 😕

Reply:
Yeah sorry, I once did something on my work pc and our internal package registry snuck into the package lock.

Reply:
Ah, that explains it! I was worried this was something malicious or something